src/resources/team/team.validator.ts

```diff
@@ -10,3 +10,10 @@ export const validateCreateTeam = () => {
 		handleValidationError,
 	]
 }
+
+export const validateGetTeams = () => {
+	return [
+		query('userId', 'User id must be a string').isString(),
```

Comment:
no need to validate user Id since we have already validated it with global middleware